docs/src/content/docs/hosting.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 'Hosting Your Own Instance'
+description: 'All you need to know to get your own Eludris instance running!'
+order: 1
+---
+
+One of our goals ever since the start is to make hosting Eludris instances as simple
+and as friction free as possible. To that extent Eludris has provided an [`Eludris.toml`](./conf.md)
+where you can configure your instance as much as you wish and also official docker
+support to help you get started scoff free.
+
+## Deploying a Production Instance With Docker
+
+We really recommend using [Docker](https://www.docker.com/) for production instances,
+it helps keep all the microservices and the external tools they rely on all in one
+neat group of containers without conflicting with anything else related to the host
+system, and without you having to go around OS-dependent installing C libraries.
+
+Deploying a production-ready Eludris instance with docker is only 2 simple steps:
+
+1. Install the [Eludris CLI](/docs/cli)
+   You can find more info about this in the [CLI docs](/docs/cli).
+
+2. Run `eludris deploy`
+   This will take you through the process of setting up your own Eludris instance step-by-step
+   without you having to worry about screwing up anywhere.
+
+And it is as simple as that.
+
+If all goes right, you can find Oprish at `0.0.0.0:7159`, Pandemonium at `0.0.0.0:7160` and Effis
+at `0.0.0.0:7161`.
+
+## Deploying on Bare Metal / Running a Development Instance
+
+Now this is just a little bit more complicated, here's a quick rundown of what you
+have to do:
+
+1. Get the required dependencies
+   You will need [KeyDB](https://docs.keydb.dev/docs/download) and [MariaDB](https://mariadb.com/downloads/),
+   the latest version of both should work.
+
+2. Get the latest version of `eludris/eludris`.
+
+```sh
+git clone https://github.com/eludris/eludris && cd eludris
+```
+
+You may also want update the submodules to use the `next` branch if you're trying
+to run a development instance or test a new feature, for that run this instead:
```

Comment:
I don't really like the flow of "for that run this instead".

Maybe something like "<...> test a new feature. To do so, run this instead:" could work?